fire/SFMainMod.F90

```diff
@@ -204,7 +189,78 @@ subroutine UpdateFuelCharacteristics(currentSite)
   end subroutine UpdateFuelCharacteristics
 
   !---------------------------------------------------------------------------------------
+  
+  subroutine CalculateIgnitionsandFDI(currentSite, bc_in)
+    !
+    !  DESCRIPTION:
+    !  Calculates ignitions and fire danger index (FDI) for a site
+    !
+    
+    use FatesInterfaceTypesMod, only : hlm_spitfire_mode
+    use EDParamsMod,            only : cg_strikes
+    use EDParamsMod,            only : ED_val_nignitions
+    use SFParamsMod,            only : SF_val_fdi_alpha
+    use FatesConstantsMod,      only : years_per_day
+
+    ! ARGUMENTS:
+    type(ed_site_type), intent(inout), target :: currentSite ! site object
+    type(bc_in_type),   intent(in)            :: bc_in       ! BC in object
+    
+    ! LOCALS:
+    type(fates_patch_type), pointer :: currentPatch            ! patch object
+    real(r8)                        :: cloud_to_ground_strikes ! fraction of cloud-to-ground strikes [0-1]
+    real(r8)                        :: anthro_ignitions        ! anthropogenic ignitions [count/km2/day]
+    integer                         :: iofp                    ! patch index
+    
+    ! CONSTANTS:
+    real(r8), parameter :: alpha = 0.0035_r8  ! potential human ignition counts (alpha in Li et al. 2012) (#/person/month)
```

Comment:
Could this be renamed to something more descriptive? E.g., `igns_per_person_month`.

Reply:
sure can do